CoroutineTemplate/app/build.gradle.kts

```diff
@@ -98,7 +98,7 @@ android {
         compose = true
     }
 
-    lintOptions {
+    lint {
```

Comment:
Can I know why we have this change? Is it because we didn't set any options? 🙇🏻‍♀️

Reply:
Ah, I found the answer [here](https://developer.android.com/reference/tools/gradle-api/7.0/com/android/build/api/dsl/LintOptions) It's incubated to `lint`. 👍🏻

Reply:
@lydiasama It was deprecated and recommended using just `lint` instead, So I updated this because we have mentioned about `lint` issue then. I think it reasonable to update it on this ticket 🙇‍♀️ 
<img width="648" alt="Screen Shot 2565-07-01 at 13 09 37" src="https://user-images.githubusercontent.com/28002315/176835009-f4bba87d-074a-46c6-8d4e-3ef0ddb84120.png">